pennylane/templates/subroutines/select.py

```diff
@@ -25,26 +25,34 @@
 
 
 class Select(Operation):
-    r"""Applies specific input operations depending on the state of
-    the designated control qubits.
+    r"""Applies individual operations depending on the state of
+    designated control qubits.
```

Comment:
I like this more but up to you :slightly_smiling_face: 
```suggestion
    the designated control qubits.
```